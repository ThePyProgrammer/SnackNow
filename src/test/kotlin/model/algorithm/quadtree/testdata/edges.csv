0,0.01,0.01,0
4,2
0,-0.01
0.01,-0.01
0,0.01
0,0.05
0,0.01,0.01,0
0,0.02,-0.02,0





