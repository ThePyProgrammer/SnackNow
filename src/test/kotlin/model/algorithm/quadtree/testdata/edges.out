Exception in thread "main" java.lang.IndexOutOfBoundsException: Point outside of bounding box of QuadTree!